### DISCUSSION TITLE
node classification datasets

### DISCUSSION THREAD

**Discussion by DuanhaoranCC** [pyg-team/pytorch_geometric#4505] [General]
Hello, I'd like to ask whether pytorch geometric provides node classification datasets in the field of biology or chemistry. I'm looking for several structure sensitive data sets. Do you have any suggestions?

Thank you very much for your patience, thank you!

**Comment by rusty1s** [comment]
I suggest you take a look at the `torch_geometric.datasets.PPI` dataset. OGB also provides the [`ogbn-proteins` dataset](https://ogb.stanford.edu/docs/nodeprop/). Note that in biology or chemistry, graph classification datasets are much more common.
